nifi-extension-bundles/nifi-snmp-bundle/nifi-snmp-processors/pom.xml

```diff
@@ -20,8 +20,8 @@ language governing permissions and limitations under the License. -->
     <packaging>jar</packaging>
 
     <properties>
-        <snmp4j.version>2.8.18</snmp4j.version>
-        <snmp4j-agent.version>2.7.9</snmp4j-agent.version>
+        <snmp4j.version>3.8.2</snmp4j.version>
+        <snmp4j-agent.version>3.8.1</snmp4j-agent.version>
```

Comment:
reviewing if changes seem safe https://www.snmp4j.org/CHANGES.txt but seems like the 2.x line was Java 8 and the 3.x line is Java 9 or higher